internal/constellation/helm/charts/edgeless/operators/charts/constellation-operator/templates/manager-rbac.yaml

```diff
@@ -13,6 +13,7 @@ rules:
   verbs:
   - get
   - list
+  - watch
```

Comment:
The RBAC config is semi-automatically generated, the source of truth is in kubebuilder comments:

https://github.com/edgelesssys/constellation/blob/76b642baf9705adddc0372339a12ddbe6af682a3/operators/constellation-node-operator/controllers/nodeversion_controller.go#L86

This is currently not enforced, but I want to canonicalize kubebuilder eventually (cf. #2837). Could you just add `watch` there for now, too?